controllers/utils/utils.go

```diff
@@ -233,6 +233,9 @@ func UpdatePodTemplateSpecFromMoverConfig(podTemplateSpec *corev1.PodTemplateSpe
 	// Security context (nil by default)
 	podTemplateSpec.Spec.SecurityContext = moverConfig.MoverSecurityContext
 
+	// Affinity
+	podTemplateSpec.Spec.Affinity = moverConfig.MoverAffinity
```

Comment:
Today, I don't think we set `Affinity` anywhere else, but I'm concerned that this assignment will overwrite the field if we do someday have an automatic/default setting for this.
What about making the assignment only in the case where the user has provided a value?

```suggestion
	if moverConfig.MoverAffinity != nil {
		podTemplateSpec.Spec.Affinity = moverConfig.MoverAffinity
	}
```

Reply:
Makes a lot of sense, ty!